bcda/bcdacli/cli.go

```diff
@@ -658,3 +680,71 @@ func setBlacklistState(cmsID string, blacklistState bool) error {
 
 	return db.Model(&aco).Update("blacklisted", blacklistState).Error
 }
+
+// CCLF file name pattern and regex
+const cclfPattern = `((?:T|P)\.BCD\..*\.ZC\d*)Y(\d{2}\.D\d{6}\.T\d{7})`
+
+var cclfregex = regexp.MustCompile(cclfPattern)
+
+func RenameCCLF(name string) string {
+	return cclfregex.ReplaceAllString(name, "${1}R${2}")
+}
+
+func CloneCCLFZips(path string) error {
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		return err
+	}
+
+	// Iterate through all cclf zip files in provided directory
+	for _, f := range files {
+		// Make sure to not clone non CCLF files in case the wrong directory is given
+		if !cclfregex.MatchString(f.Name()) {
+			log.Infof("Skipping file `%s`, does not match expected name... ", f.Name())
+			continue
+		}
+		fn, err := CloneCCLFZip(path, f.Name())
+		if err != nil {
+			return err
+		}
+		log.Infof("Created runout file: %s", fn)
+	}
+	return nil
+}
+
+func CloneCCLFZip(path, name string) (string, error) {
```

Comment:
nit - would function make more sense with the `(src, dest string)` args? It would make this look more like io.Copy(). So it could be called with something like this: `paths.Join(path, f.Name()), paths.Join(path, RenameCCLF(f.Name())`

Reply:
Do you see value in having this available outside of the cli package? If not, I think we should not export it and make it cloneCCLFZip.